### DISCUSSION TITLE
Ansible Galaxy?

### DISCUSSION THREAD

**Discussion by Andyjb8** [jeisenbath/ansible-collection-solarwinds-orion#13] [Q&A] [answered]
Is there a reason this collection is not in ansible galaxy? It might get more visibility there.

**Comment by jeisenbath** [comment]
Yeah mostly because I haven't had time to work on getting it there. I have a bit of an internal checklist to get done before trying to publish. Getting the changelog figured out was one, and that's done. Getting the doc RST files created is another, then updating the README with some contributing guidelines (such as changelog requirements). Then I'd like to publish it to the Solarwinds namespace, and not my personal one (or submit it to the community namespace, so it would be community.solarwinds likely instead of solarwinds.orion), so I need to figure out how to do that.

Basically, all boils down to time. I also only in the last few months unforked the repo, which was another thing on the checklist. A couple of big projects at work have me pushing this off, but the bright side is one of those projects has me writing a dynamic inventory plugin, so hopefully I can take that experience to create a Solarwinds dynamic inventory plugin also!

**Comment by jeisenbath** [comment, marked as answer]
Collection is published! The namespace for all modules, plugins, and roles have migrated with the 3.0.0 version from solarwinds.orion to jeisenbath.solarwinds, you can install the 3.0.0 version with 'ansible-galaxy collection install jeisenbath.solarwinds', and you'll want to update your playbooks, roles, and inventory to the new namespace. Any new features I only plan to include in >3.0.0 releases, but I'll try to keep porting bugfixes to the stable-2.x branch. If you are still using the old namespace, you can keep that up to date with 'ansible-galaxy collection install git+https://github.com/jeisenbath/ansible-collection-solarwinds-orion.git,stable-2.x'
